### PR TITLE
support edit cluster template

### PR DESCRIPTION
**Pull request by batzionb**
fixed details section in details page to match UX changes
disabled switching to wizard steps after current wizard s

### CODE REVIEW

File: src/components/ClusterTemplateWizard/ClusterTemplateWizard.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  return (
+    <Loader loaded={initialValuesLoaded && loadedValidationSchema} error={initialValuesError}>
+      <Formik<WizardFormikValues>
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+        validationSchema={validationSchema}
+      >
+        <Wizard
+          steps={steps}
+          footer={<CustomFooter onNext={(stepId) => console.log('footer on next', stepId)} />}
```
Comment (jtomasek) [inline src/components/ClusterTemplateWizard/ClusterTemplateWizard.tsx:184]: The `console.log` call is a leftover?

File: console-extensions.json [hunk 2]
```diff
@@ -114,5 +114,29 @@
...
+      "id": "admin-cluster-templates-console",
+      "perspective": "admin",
+      "section": "compute",
+      "name": "%plugin__clustertemplates-plugin~Cluster templates%",
+      "href": "/k8s/cluster/clustertemplate.openshift.io~v1alpha1~ClusterTemplate",
+      "insertAfter": "nodes"
+    },
+    "flags": {
+      "required": ["CLUSTER_TEMPLATES_FLAG"]
+    }
```
Comment (jtomasek) [inline console-extensions.json:140]: This should not be included but let's keep it for now for a simpler dev experience :)

File: src/components/ClusterTemplateWizard/WizardPage.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const WizardPage = () => {};
```
Comment (jtomasek) [inline src/components/ClusterTemplateWizard/WizardPage.tsx:1]: Unused, should be removed.

**Review by jtomasek — COMMENTED** [review]
The helper components should move from `src/helpers` to `src/components/ui`

**Review by jtomasek — APPROVED** [review]

File: src/components/ClusterTemplateWizard/ClusterTemplateWizard.tsx (hunk 1 above)
Comment (batzionb) [reply src/components/ClusterTemplateWizard/ClusterTemplateWizard.tsx:184]: yes, I'll remove it

File: console-extensions.json (hunk 2 above)
Comment (batzionb) [reply console-extensions.json:140]: I'll make sure to remember to remove it before it's released :)

**Pull request merged**